quarkus/container/Dockerfile

```diff
@@ -19,6 +19,9 @@ RUN chmod -R g+rwX /opt/keycloak
 
 FROM registry.access.redhat.com/ubi8-minimal
 
+RUN microdnf install glibc-langpack-en; sed -i 's/^LANG=.*/LANG="en_US.utf8"/' /etc/locale.conf
+ENV LANG en_US.UTF-8
+
 COPY --from=build-env --chown=1000:0 /opt/keycloak /opt/keycloak
```

Comment:
Here you are adding a dependency and changing things in the "builder" image, those changes won't affect the "resulting" image.
Am I missing something?